RationaleMCP/0031/concept.md

```diff
@@ -0,0 +1,135 @@
+# Concept of Flat Modelica
+
+Flat Modelica is designed to be a standardized, intermediate file format which can be produced in the middle of the transformation process from a Modelica model to an executable model. 
```

Comment:
To me, just saying _can be produced_ misses the point of being a true intermediate format.  To me, the central idea of being an intermediate format is that it should be a smaller language to which full fledged Modelica is reduced without loss of any of the information needed to complete the translation.  This is the key to separating the complicated high level constructs of Modelica from the essence of solving the equations.

Reply:
Ok, this needs to be reformulated...

Reply:
How about:
_Flat Modelica is designed to be a minimalistic language to describe a hybrid DAE systems, serving as a standardized intermediate format in the transformation process from a full fledged object-oriented Modelica model to an executable simulation model._

Reply:
Didn't we already settle this at the meeting two weeks ago?  It ended up [here](https://github.com/modelica/ModelicaSpecification/tree/MCP/0031/RationaleMCP/0031#in-a-sentence-or-two).